src/layout/Header/Header.tsx

```diff
@@ -1,9 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { LANGUAGES } from '../../config/constants';
 import { HeaderWrapper } from './Header.styled';
 
 export const Header = () => {
+    const { i18n, t } = useTranslation();
+
+    const onChangeLang = (e: React.ChangeEvent<HTMLSelectElement>) => {
+        const lang_code = e.target.value;
+        i18n.changeLanguage(lang_code);
```

Comment:
`changeLanguage` seems to be a promise. Please consider handling promises using `.then` and `.catch` or async await with try catch block.

Reply:
Resolved in 6340f9e23c49c36c04ec0737e94041cf57b10289